pkgs/ok_http/lib/src/ok_http_client.dart

```diff
@@ -53,6 +53,9 @@ class OkHttpClient extends BaseClient {
   @override
   void close() {
     if (!_isClosed) {
+      // Refer to OkHttp documentation for the shutdown procedure:
+      // https://square.github.io/okhttp/5.x/okhttp/okhttp3/-ok-http-client/index.html#:~:text=Shutdown
```

Comment:
Shouldn't you also clear the cache?

Reply:
I was unsure of it, since I read that
> If your client has a cache...

I'll go ahead and add that

Reply:
Done! ✅